XYZTranslation/XYZTranslationTests.c

```diff
@@ -65,6 +87,7 @@ void dirtyTestTwo() {
   setYTranslation(FLT_MIN * 2);
   setZTranslation(FLT_MIN * 2);
   xyzTranslation();
+  printf("Logical Error: Size out of bounds\n");
```

Comment:
Logical Error should not be printed during tests. only failing tests should have prints